src/browser_action/vitals.js

```diff
@@ -105,23 +105,32 @@
           overlayClose.innerText = 'Close';
           overlayClose.id = 'web-vitals-close';
           overlayClose.className = 'lh-overlay-close';
-          overlayClose.addEventListener('click', () => {
-            overlayElement.remove();
-            overlayClose.remove();
-            overlayClosedForSession = true;
-          });
+
           document.body.appendChild(overlayClose);
-        } else {
-          overlayClose.addEventListener('click', () => {
-            overlayElement.remove();
-            overlayClose.remove();
-            overlayClosedForSession = true;
-          });
         }
+
+        overlayClose.addEventListener('click', () => {
+          closeOverlay(overlayElement, overlayClose);
+        });
       }
     });
   }
 
+  /**
+ * Closes the overlay and disables it from the options menu
+ * @param {Element} overlayElement
+ * @param {Element} overlayCloseElement
+ */
+
+  function closeOverlay(overlayElement, overlayCloseElement) {
+    overlayElement.remove();
+    overlayCloseElement.remove();
+
+    chrome.storage.sync.set({
+      enableOverlay: false,
```

Comment:
This disables the overlay for every tab, but I think the intention was to make it hidden only for the current tab.

Reply:
Hmmm thought the idea was to hide it entirely, but probably makes more sense to hide it just for the current tab.

@addyosmani if that's the case, how would you like the `Options` page to handle it? Should overlay preferences also be able to control on a per-tab basis? 

Maybe something like this?

![Screen Shot 2020-05-04 at 3 33 13 PM](https://user-images.githubusercontent.com/12476932/81005893-a663ac80-8e1c-11ea-996a-60e7b24bd0e1.png)

Reply:
I think your proposal to expose an option to display HUD overlay on current tab sgtm.

Reply:
So that opens a whole can of worms. Not as easy as I thought it would be to handle displaying the HUD on a single tab vs. all tabs) :(

But I did update the PR to at least limit "close" to disabling just the single tab!